packages/instrumented-component-lifecycle/tests/instrumented-component-lifecycle.test.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import { render, screen } from '@testing-library/react'
+import { withInstrumentedComponentLifecycle } from '../lib/instrumented-component-lifecycle'
```

Comment:
This also tests the configuration of the `package.json`

```suggestion
import { withInstrumentedComponentLifecycle } from '../'
```